app/views/placements/schools/mentors/remove.html.erb

```diff
@@ -7,19 +7,23 @@
 <div class="govuk-width-container">
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <label class="govuk-label govuk-label--l">
-        <span class="govuk-caption-l"><%= @mentor.full_name %></span>
-        <%= t(".are_you_sure") %>
-      </label>
+      <% if policy(@mentor_membership).destroy? %>
```

Comment:
Not a biggie, but I'd be tempted to move this logic up into the controller and render different views accordingly.

```ruby
def remove
  if policy(@mentor_membership).destroy?
    render "confirm_remove"
  else
    render "can_not_remove"
  end
end
```

That way it's really obvious from looking at the controller that the page behaves differently depending on the circumstances. And then the view becomes more "dumb" rather than logic hiding in it.